src/main.rs

```diff
@@ -0,0 +1,169 @@
+use typedb_client::{
```

Comment:
Let's have a licence header at the top of this file too.

Reply:
Got it. Done